demo/components-mix-demo.html

```diff
@@ -0,0 +1,296 @@
+                --px-tile-subtitle-color: #2c404c;
+                --px-tile-title-color: #121f26;
+            }
+            .px-tile-custom {
+              --px-tile-height: 10rem;
+              --px-tile-background-color: #B6C3CC;
+              --px-tile-about-min-height: 10rem;
+              --px-tile-action-buttons-position: absolute;
+              --px-tile-action-buttons-right: 15px;
+              --px-tile-action-buttons-bottom: 15px;
+            }
+            .small-container {
+              --px-tile-height: 5rem;
+              --px-tile-about-min-height: 5rem;
+            }
+            /* square sample  */
+            .square-container {
+              --px-tile-height: 135px;
+              --px-tile-width: 135px;
+              --px-tile-about-min-height: 25px;
+              --px-tile-about-width: 25px;
+              --px-tile-about-padding: 0px;
+              --px-tile-about-right: 0px;
+              --px-tile-text-background-color: #D8E0E5;
+              
+              --px-tile-overlay-action-buttons-position: absolute;
+              --px-tile-overlay-action-buttons-bottom: -12px; 
+              --px-tile-overlay-action-buttons-right: -14px;
+              --px-tile-overlay-width: 25px;
+            }
+            .center-text {
+              height: 135px;
+              width: 135px;
+              display: flex;
+					    align-items: center;
+					    justify-content: center;
+            }
+            
+            
+            .footer {
+                display: flex;
+                align-items: center;
+                margin-left: 15px;
+            }
+            px-alert-label {
+                margin-left: 0.66667rem;
+                --px-alert-label-border-color: white;
+                --px-alert-label-background-color--unknown: transparent;
+                --px-alert-label-text-color--unknown: white;
+            }
+
+            .px-simple-horizontal-bar-chart {
+                padding: 1rem 0.5rem 1rem 1rem;
+            }
+        </style>
+    </custom-style>
+
+    <link rel="import" href="../px-tile.html" />
+
+    <script>
+        window.addEventListener('px-title-on-action-clicked', function(evt){
+          console.log(evt.detail); 
+          var msg = document.getElementById('selection-mssg');
+          var el = document.getElementById('selection');
+          if(msg && el) {
+            msg.innerText = 'You have selected: ';
+            el.innerText = JSON.stringify(evt.detail);
+          }
+        }); 
+    </script>
+
+</head>
+
+<body>
+
+<style>
+  .info-container {
+    display: flex;
+  }
+  .info-item {
+    padding: 20px;
+  }
+	.thumbnail{
+	    text-align: center;
+	}
+	.info {
+	  padding: 20px;
+	}
+	.selection {
+	  font-size: 22px;
+	}
+	.code-sample {
+	  min-width: 520px;
+	}
+</style>
+<div class="info">
+  https://www.predix-ui.com
+</div>
+
+<div style="display:flex; flex-wrap:wrap;">
+    <px-tile 
+             title="02-88583"
```

Comment:
This actually would have been introduced in the last PR, but I've just remembered why `title` is problematic as a property name, since it's already a built-in HTML behavior: https://github.com/PredixDev/px-card/issues/3

We should reconsider that.